src/fragments/lib/ios-prereq-xcode.mdx

```diff
@@ -1 +1,5 @@
-* [Xcode](https://developer.apple.com/download/applications/) version 14.1 or later
\ No newline at end of file
+* For iOS, [Xcode](https://developer.apple.com/download/applications/) version 14.1 or later
```

Comment:
Isn't this requirement only mentioned when building specifically an **iOS** app?

Reply:
This fragment is used at two places - 
getting started - it is explicitly mentioned for iOS
project setup - it is not explicitly mentioned but in the following steps, an iOS app is being created.
I will update the project setup wording and remove this change.

Reply:
Updated screenshot:

<img width="539" alt="Screenshot 2024-01-24 at 14 26 41" src="https://github.com/aws-amplify/docs/assets/23294582/6d8ea5b5-ea39-4abf-886e-fd40d814d606">